config/application.rb

```diff
@@ -52,6 +52,9 @@ class Application < Rails::Application
 
     config.paths["log"] = ENV["LOG_PATH"] if ENV["LOG_PATH"]
 
+    # Google Analytics ID
+    config.universalId = ENV["GA_UNIVERSAL_ID"]
```

Comment:
Nitpick - the convention for variables is to use snake case.

Reply:
Also can we set this as a default value here and then only override the one in "environments/production.rb" rather than needing entries in config/development and config/test ?